test/tests/functional/pbs_eligible_time.py

```diff
@@ -110,15 +110,42 @@ def test_job_array(self):
                            extend='t', offset=20)
         self.server.expect(JOB, {'accrue_type': 1}, id=jid)
 
-        # Capture the time stamp when accrue type changed to
-        # ineligible time
-        msg2 = jid + ";Accrue type has changed to ineligible_time"
+        # Capture the time stamp when subjob 3 starts run. Accrue type changes
+        # to ineligible time. eligible_time calculation is completed.
+        msg2 = J1.create_subjob_id(jid, 3) + ";Job Run at request of Scheduler"
         m2 = self.server.log_match(msg2)
         t2 = logutils.convert_date_time(m2[1].split(';')[0])
         eligible_time = t2 - t1
 
-        m = jid + ";Accrue type has changed to ineligible_time, "
-        m += "previous accrue type was eligible_time"
-        m += " for %d secs, " % eligible_time
-        m += "total eligible_time=00:00:%d" % eligible_time
-        self.server.log_match(m)
+        m1 = jid + ";Accrue type has changed to ineligible_time, "
+        m1 += "previous accrue type was eligible_time"
+
+        m2 = m1 + " for %d secs, " % eligible_time
+        m2 += "total eligible_time=%s" % time.strftime(
+              "%H:%M:%S", time.gmtime(eligible_time))
+        try:
+            self.server.log_match(m2)
+        except PtlLogMatchError as e:
+            # In some test bed machines, there is a delay observed between
```

Comment:
Instead of saying testbed machines, say slow machines.

Reply:
Done